html/browsers/windows/nested-browsing-contexts/resources/post-to-parent.html

```diff
@@ -0,0 +1,6 @@
+<script>
+const ident = new URL(location).searchParams.get("ident"),
+      post = () => parent.postMessage({ name, ident }, "*");
```

Comment:
It's a little unclear that this is equivalent to `name: window.name` so I'd suggest changing it to make it explicit.